clients/java/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'java'
+}
+
+group 'signalr'
```

Comment:
We need to decide on our groupId (a "namespace" concept in Maven). Azure SDKs are under `com.microsoft.azure`, and there are similar org-specific hives under `com.microsoft`: https://mvnrepository.com/artifact/com.microsoft . I think `com.microsoft.aspnet` or `com.microsoft.aspnetcore` would work.

Also, it looks like the artifactId (Maven's packageID equivalent, but scoped within the groupId) is taken from the folder name by default, meaning this one will be `java`, so our full package ID right now is `signalr:java`. Not sure that's what we want ;P. Since we probably need samples too, we could create a folder structure like: `clients/java/signalr` (for the client) and `clients/java/signalr-client-sample` for a sample client?

MyGet supports publishing Maven packages, so we can do some test publishing to a rando MyGet feed to make sure we're happy with all those bits and bobs.